wai-aria/role/abstract-roles.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+<head>
+  <title>Abstract Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+
+<p>Verifies <a href="https://w3c.github.io/aria/#document-handling_author-errors_roles">9.1 Roles - handling author errors</a></p>
```

Comment:
This PR now closes:

- https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/12